packages/hebao_v1/contracts/base/BaseSubAccount.sol

```diff
@@ -139,7 +139,13 @@ contract BaseSubAccount is SubAccount
         }
     }
 
-    function tokenInterestRate(address, address, address) public view returns (int) { return 0; }
+    function tokenInterestRate(address, address, address)
```

Comment:
The latest parameter needs to be `uint` for the amount it seems.

Reply:
already changed in #822

Reply:
OK, I'll cancel this PR then.